go/vm/types.go

```diff
@@ -29,8 +32,42 @@ func (a *Address) UnmarshalText(data []byte) error {
 	return textToBytes(a[:], data)
 }
 
+func (v Value) ToBig() *big.Int {
+	return new(big.Int).SetBytes(v[:])
+}
+
+func (v Value) ToUint256() *uint256.Int {
+	return new(uint256.Int).SetBytes(v[:])
+}
+
 func (v Value) String() string {
-	return fmt.Sprintf("0x%x", v[:])
+	return v.ToUint256().String()
+}
+
+func (v Value) Cmp(o Value) int {
+	return v.ToUint256().Cmp(o.ToUint256())
```

Comment:
does this means 2 copies for every call to `Cmp` ?

Reply:
Yes, I went for the simple solution and would have looked into performance issues as they are showing up. But it is easy to fix, to get rid of the copies. I made the corresponding change.